lib/disclosure/index.d.ts

```diff
@@ -0,0 +1,3 @@
+import initDisclosure from './disclosure';
```

Comment:
Haven't seen these `.d.ts` files before. Here's a helpful [reddit comment](https://www.reddit.com/r/typescript/comments/hly19w/help_me_understand_dts_files/) on them for anyone interested

Reply:
Oh good spot, these should actually go as they are the old generated typescript output